README.md

```diff
@@ -28,6 +28,16 @@ For Corda, we build Docker containers from the Corda source. A number of differe
 
 ![Blockchain Automation Framework - Corda](./docs/images/blockchain-automation-framework-corda.png "Blockchain Automation Framework for Corda")
 
+## Build
```

Comment:
Is this section specifically related to Corda? If so, this should probably be ### heading. If not, do we need to update line 38, which seems Corda specific?